### PR TITLE
Enterprise 4.2 monitoring metrics ui

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@ahardin-rh Could you review this please? If it looks good, it should go to master too.

**Comment by vikram-redhat** [comment]
@rh-max can you close this and set against master first?

**Comment by rh-max** [comment]
@vikram-redhat As discussed with you, keeping this, to be merged to master in the end.

**Comment by rh-max** [comment]
@jboxman Hi Jason, could you please review, merge & cherry-pick to enterprise-4.2? Thanks!

**Comment by rh-max** [comment]
That is, cherry-pick to master (this is enterprise-4.2).

File: monitoring/cluster-monitoring/managing-cluster-alerts.adoc [hunk 1]
```diff
@@ -16,5 +16,5 @@ include::modules/monitoring-expiring-silences.adoc[leveloffset=+1]
 
 .Next steps
 
-xref:../../monitoring/cluster-monitoring/prometheus-alertmanager-and-grafana.adoc#prometheus-alertmanager-and-grafana[Access the Prometheus, Alertmanager, and Grafana.]
+xref:../../monitoring/cluster-monitoring/examining-cluster-metrics.adoc#examining-cluster-metrics[Examine cluster metrics.]
```
Comment (jboxman) [inline monitoring/cluster-monitoring/managing-cluster-alerts.adoc]: I'd put the period after the link instead of as part of it.

**Comment by jboxman** [comment]
@rh-max, ideally we'd create this PR against `master`, and then cherry-pick it into `enterprise-4.2`.

**Comment by jboxman** [comment]
@rh-max, per our OpenShift docs contributing guidelines, we're wrapping text at 80 columns:

https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

**Comment by rh-max** [comment]
Closing this in favor of #16752 .

**Pull request closed without merging**